doc/rados/operations/stretch-mode.rst

```diff
@@ -8,110 +8,114 @@ Stretch Clusters
 Stretch Clusters
 ================
 
-A stretch cluster is a cluster that has servers in geographically separated
-data centers, distributed over a WAN. Stretch clusters have LAN-like high-speed
-and low-latency connections, but limited links. Stretch clusters have a higher
+A stretch cluster spreads hosts across geographically separated
+data centers, with specific internal strategies to robustly localize
+network traffic and respond to failures.
+Stretch clusters have a higher
 likelihood of (possibly asymmetric) network splits, and a higher likelihood of
 temporary or complete loss of an entire data center (which can represent
 one-third to one-half of the total cluster).
 
 Ceph is designed with the expectation that all parts of its network and cluster
 will be reliable and that failures will be distributed randomly across the
-CRUSH map. Even if a switch goes down and causes the loss of many OSDs, Ceph is
+CRUSH topology. If a host or network switch goes down and causes the loss of many OSDs, Ceph is
```

Comment:
At some future time, I think that the sentence that here begins with "If a host" should be rewritten so that its grammar does not suggest that the design of Ceph is conditional upon the loss of a host or network switch.  It's good enough to merge as it is, though, and I will not hold up the merging and backporting of this PR on account of this one sentence.